Configuration/PyReleaseValidation/python/relval_nano.py

```diff
@@ -359,6 +365,7 @@ def subnext(self):
 
 # POG/PAG custom NANOs, MC
 _wfn.subnext()
+workflows[_wfn()] = ['BPHNANOmc140X', ['TTbarMINIAOD14.0', 'BPHNANO_mc14.0']]
```

Comment:
Looks like the workflow ids were shift by +1 when the BOPHNano workflow was added. This also shift the base x ref comparisons,  eg.: `EGMNANOmc140X` was 2500.222 and now it is 2500.223,   so `EGMNANOmc140X` is being compared with `BTVNANOmc140X` (the previous 2500.223).

@gmelachr -- can you try to add `BPHNANOmc140X` at the end of the `subnext` block?